dev/io.openliberty.microprofile.health.internal.common/resources/io/openliberty/microprofile/health/resources/Health.nlsprops

```diff
@@ -79,6 +79,18 @@ file.healthcheck.file.delete.fail.CWMMH0106W=CWMMH0106W: Unable to delete existi
 file.healthcheck.file.delete.fail.CWMMH0106W.explanation=The file-based health checks require the creation and use of the files "live", "ready" and "started" in the /health directory. For the file-based health checks to function correctly, these files must not exist when the runtime starts. The runtime has attempted to delete the existing file was unable to.
 file.healthcheck.file.delete.fail.CWMMH0106W.useraction=The user must identify the root cause preventing the runtime from deleting the files.
 
+file.healthcheck.system.inconsistency.CWMMH0107W=CWMMH0107W: The runtime has detected inconsistencies with the local file-based health check mechanism. The file-based health check function will be disabled.
+file.healthcheck.system.inconsistency.CWMMH0107W.explanation= Inconsistencies were detected with the file system. For example, the "started" file is present before it was created by the runtime.
+file.healthcheck.system.inconsistency.CWMMH0107W.useraction=Delete the <server.output.dir>/health directory and restart the server.
+
 file.update.interval.config.invalid.CWMMH01010W=CWMMH01010W: An invalid value [{0}] was provided for the file update interval. The value provided must consist of a number followed by an optional time unit of "ms" for milliseconds or "s" for seconds. If no time unit is specified, the value is in seconds by default. The runtime will default to 10 seconds.
 file.update.interval.config.invalid.CWMMH01010W.explanation=The file update interval configuration only accepts a number followed by an optional time unit of "ms" for milliseconds or "s" for seconds. If no time unit is specified, the value is in seconds by default.
-file.update.interval.config.invalid.CWMMH01010W.useraction=Input a correct value for the file update interval configuration.
\ No newline at end of file
+file.update.interval.config.invalid.CWMMH01010W.useraction=Input a correct value for the file update interval configuration.
+
+file.create.interval.config.invalid.CWMMH01011W=CWMMH01011W: An invalid value [{0}] was provided for the file create interval. The value provided must consist of a number followed by an optional time unit of "ms" for milliseconds or "s" for seconds. If no time unit is specified, the value is in milliseconds by default. The runtime will default to 100 milliseconds.
+file.create.interval.config.invalid.CWMMH01011W.explanation=The file create interval configuration only accepts a number followed by an optional time unit of "ms" for milliseconds or "s" for seconds. If no time unit is specified, the value is in milliseconds by default.
+file.create.interval.config.invalid.CWMMH01011W.useraction=Input a correct value for the file create interval configuration.
+
+file.create.interval.config.only.set.CWMMH01012W=CWMMH01012W: The file create interval configuration was set without the file update interval configuration set. The file update interval is required to be configured to enable the file-based health check.
```

Comment:
Change to:
file.create.interval.config.only.set.CWMMH01012W=CWMMH01012W: The file create interval configuration was set before the file update interval was configured. The file-based health check is disabled when the file update interval is not configured.
file.create.interval.config.only.set.CWMMH01012W.explanation=Configuring only the file create interval does not enable the file-based health check mechanism. 
file.create.interval.config.only.set.CWMMH01012W.useraction=Configure the file update interval to enable the file-based health check mechanism.